javascript/MaterialXView/source/helper.js

```diff
@@ -229,26 +230,33 @@ function getMinFilter(type, generateMipmaps)
  * @param {mx.Uniforms} uniforms
  * @param {mx.TextureFilter.generateMipmaps} generateMipmaps
  */
-function setTextureParameters(texture, name, uniforms, flipY = true, generateMipmaps = true)
-{
-    const idx = name.lastIndexOf(IMAGE_PROPERTY_SEPARATOR);
-    const base = name.substring(0, idx) || name;
-
-    texture.generateMipmaps = generateMipmaps;
-
-    const uaddressmode = uniforms.find(base + UADDRESS_MODE_SUFFIX)?.getValue().getData();
-    const vaddressmode = uniforms.find(base + VADDRESS_MODE_SUFFIX)?.getValue().getData();
-
-    texture.wrapS = getWrapping(uaddressmode);
-    texture.wrapT = getWrapping(vaddressmode);
-
-    const filterType = uniforms.get(base + FILTER_TYPE_SUFFIX) ? uniforms.get(base + FILTER_TYPE_SUFFIX).value : -1;
-    texture.magFilter = THREE.LinearFilter;
-    texture.minFilter = getMinFilter(filterType, generateMipmaps);
-
-    texture.flipY = flipY;
-}
-
+ function setTextureParameters(texture, name, uniforms, flipY = true, generateMipmaps = true)
```

Comment:
Small changes to make sure to handle if uniforms accessed here exist as they may not
if using reduced mode.